tools/mcp/dotnet/AzureSDKDevToolsMCP/Program.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureSDKDevToolsMCP.Tools;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using ModelContextProtocol.Server;
+
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddMcpServer().WithToolsFromAssembly();
+var app = builder.Build();
+
+// 👇 Map Mcp endpoints
+app.MapMcp();
+
+app.Run();
```

Comment:
Does this support stdio mode or just SSE right now? For both I have some common arguments in mind (stdio default, a flag to specify SSE transport, and to specify port).

Reply:
Just SSE for now. But I want to add stdio with a static executable to make it easier for users.